app/repos/police-forces.js

```diff
@@ -90,9 +90,33 @@ const getPoliceForceByShortName = async (shortName, transaction) => {
   })
 }
 
+const extractShortNameAndDomain = (email) => {
+  const lowerEmail = email.toLowerCase()
+  const atPos = lowerEmail.indexOf('@')
+  if (atPos === -1) {
+    return email
+  }
+  const domain = email.substr(atPos + 1)
+  const shortName = domain.replace('.pnn.police.uk', '').replace('.police.uk', '')
```

Comment:
this code is kind of duplicated - app/repos/user-accounts.js line 68

Reply:
So it is. I'll move it out to a common method